packages/rat-king-parser.vm/tools/chocolateyinstall.ps1

```diff
@@ -0,0 +1,8 @@
+$ErrorActionPreference = 'Stop'
+Import-Module vm.common -Force -DisableNameChecking
+
+$toolName = 'rat-king-parser'
+$category = 'Packers'
```

Comment:
I think Packers is not the correct category here, opinions @mandiant/flare-vm?
```suggestion
$category = 'Utilities'
```

@sara-rn you can edit the issue and then re-trigger the automation **without creating a new issue**. Let me know if this is not clear.